src/class-parsely.php

```diff
@@ -129,6 +131,52 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	/**
+	 * Include a link to the statistics page for an article in the wp-admin Posts List.
+	 * If the post object is the "front page," this will include the main dashboard link instead.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/page_row_actions/
+	 * @see https://developer.wordpress.org/reference/hooks/post_row_actions/
+	 * @param [string[]] $actions  The list of actions to which this function will append
+	 * @param [WP_Post]  $post     The individual post object in the list
+	 * @return $actions            The filtered list of actions
+	 */
+	public function row_actions_add_parsely_link( $actions, $post ) {
+		if ( ! is_array( $actions ) ) {
+			return $actions;
+		}
+
+		if ( ! self::post_has_trackable_status( $post ) ) {
+			return $actions;
+		}
+
+		$options = $this->get_options();
+		$apikey = $options['apikey'];
+		$parsely_url = trailingslashit( 'https://dash.parsely.com/' . $apikey );
```

Comment:
Should we check that apikey has been set before assuming it has? Can't create a link if we don't know where we're meant to link to.